src/components/switch.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css, Input } from 'theme-ui';
+
+const defaultVariants = {
+  sm: {
+    width: '30px',
+    height: '15px',
+    border: '#ddd',
+    bg: '#ddd',
+    color: '#fff',
+  },
+  md: {
+    width: '38px',
+    height: '19px',
+    border: '#ddd',
+    bg: '#ddd',
+    color: '#fff',
+  },
+  lg: {
+    width: '44px',
+    height: '22px',
+    border: '#ddd',
+    bg: '#ddd',
+    color: '#fff',
+  },
+};
+
+const getVariants = (theme, variant = 'md', key) =>
+  theme.switch[variant] && theme.switch[variant][key]
+    ? theme.switch[variant][key]
+    : defaultVariants[variant][key];
```

Comment:
@a-saran 
- [ ] Can't you make use of `applyVariations` method you had written before?

Reply:
we are not using applyVariations here?